Tests/SentryTests/SentryHubTests.swift

```diff
@@ -204,6 +206,90 @@ class SentryHubTests: XCTestCase {
         }
     }
     
+    func testStartTransactionWithNameOperation() {
+        let span = fixture.getSut().startTransaction(name: fixture.transactionName, operation: fixture.transactionOperation)
+        let tracer = Dynamic(span)
+        XCTAssertEqual(tracer.name, fixture.transactionName)
+        XCTAssertEqual(span.context.operation, fixture.transactionOperation)
+    }
+    
+    func testStartTransactionWithContext() {
+        let span = fixture.getSut().startTransaction(transactionContext: TransactionContext(name: fixture.transactionName, operation: fixture.transactionOperation))
+        
+        let tracer = Dynamic(span)
+        XCTAssertEqual(tracer.name, fixture.transactionName)
+        XCTAssertEqual(span.context.operation, fixture.transactionOperation)
```

Comment:
```
et tracer = Dynamic(span)
        XCTAssertEqual(tracer.name, fixture.transactionName)
        XCTAssertEqual(span.context.operation, fixture.transactionOperation)
```

duplicated blocks could be a method called `assertSpan` that takes tracer and span as a method param, fixture is already visible, so we save DRY